### ISSUE TITLE
🐛 [BUG] - <Some Items in database list do not have a popup window >

### ISSUE THREAD

**Issue by ktha423** [SE310-Team1/ShoppIt#75]
### Description

Some items such as carrot, spring onions and mushroom do not open a new popup window which lets us add it to the list. 

### Reproduction URL

https://github.com/SE310-Team1/ShoppIt

### Reproduction steps

```bash
1. Go to 'SearchPopUpScene'
2. Click on 'carrots' 
3. See error
```

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

### OS

Windows

**Comment by poli648** [comment]
Yeah, that is a crucial issue that needs to be solved. Not being able to buy carrots, and other items are preventing this app from being great. Do we have any ideas on why this bug is occurring?

**Comment by ktha423** [comment]
I think the image files are wrong so might be causing it to not load up

**Comment by poli648** [comment]
Ah yes, what a keen observation, great job! This needs to be sorted out ASAP!

**Comment by ktha423** [comment]
closed by pull request #87
